scripts/TestHarness/testers/RavenUtils.py

```diff
@@ -308,10 +361,6 @@ def parse_conda_for_os(libs, op_sys):
   if '--conda-forge' in sys.argv:
     # just install command is generated
     condaForge = True
-  if '--py3' in sys.argv:
```

Comment:
I really don't like using the environment variable instead of a command-line argument to select the python version. I think we agreed on not pursuing the command line removal in a team discussion, right?

Reply:
not valid